demo/README.md

```diff
@@ -0,0 +1,55 @@
+Here are demo scenarios you can use to evaluate Robusta.dev. Each demo can be run with a single bash command.
+
+# Crashing Pod Demo
+
+## Running it
+
+1. Run the script `./crash_pod_demo/crashpod_demo.sh`
+2. Press enter to deliberately break the `crashpod` deployment and make it crash
+3. Check your Slack channel for a notication about the crashing pod. The message includes the pod's logs!
+4. Click the "Investigate button" to open Robusta's UI
+5. Click on the last change event and see exactly what broke the pod
+
+## Value Demonstrated
+
+This demo shows:
+
+1. You can get Slack notifications about issues in your cluster 
+2. Notifications contain the exact data you need to debug
+3. You see more troubleshooting data in the Robusta UI, including recent changes to Kubernetes objects
+
+# OOM Kill Demo
+
+## Running it
+
+1. Run `kubectl apply -f ./oomkill_demo/pod_oomkill.yaml`
+2. Check your Slack channel for a notification about the crashing pod. The message includes details you need to debug the OOM
```

Comment:
Is this the crashing pod or oom killed pod?

Reply:
What do you mean?

Reply:
It's oom kill demo, and the notification should be oom kill enricher, right?
The text says "for a notification about the crashing pod"
Should this be "for a notification about the oom killed pod" ?